styles/styles.css

```diff
@@ -50,5 +50,59 @@ ol::after {
     /*height: 100px;*/
 }
 .button {
-    min-width: 70px;
+    min-width: 70px
+}
+/*Lesson 4 */
+.advantage {
+    max-width: 1480px;
+    margin: 0 auto;
+}
+
+.advantage__title {
+    text-align: center;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 40px;
+    line-height: 90%;
+    color: #333333;
+    margin-bottom: 100px;
+}
+
+.advantage__list {
+    list-style: none;
+}
+
+.advantage__item {
+    display: inline-block;
+    max-width: 300px;
+    margin-right: 200px;
+    vertical-align: top;
+}
+
+.advantage__item:last-child {
+    margin-right: 0;
+}
+
+.advantage__desc {
+    font-weight: 400;
+    font-size: 24px;
+    line-height: 100%;
+    text-align: left;
+    margin: 50px 0;
+    color: #333;
+}
+
+.advantage__block {
+    height: 180px;
+    width: 200px;
+    margin: 0 auto 32px;
+}
+
+.advantage__icon {
+
+}
+
+img {
+    height: 100px;
```

Comment:
Better to remove property height
add
 max-width:100px
width:auto
It will be look better and more flexible